core/tests/test_utils.py

```diff
@@ -322,6 +326,89 @@ def generate_random_hexa_str() -> str:
     return ''.join(random.choices(uppercase + lowercase + string.digits, k=32))
 
 
+@contextlib.contextmanager
+def swap_is_feature_flag_enabled_function(
+    feature_flag_names: List[feature_flag_list.FeatureNames]
+) -> Iterator[None]:
+    """Mocks is_feature_flag_enabled function within the context of a
+    'with' statement. is_feature_flag_enabled will return True for all
+    the features present in feature_flag_names.
+
+    Args:
+        feature_flag_names: List[FeatureNames]. The name of the feature
+            flags for which the value should be returned as True.
+
+    Yields:
+        context. The context with function replaced.
+    """
+    def mock_is_feature_flag_enabled(
+        _: str, target_feature_flag_name: str
+    ) -> bool:
+        """Mocks is_feature_flag_enabled function to return True if the
+        target_feature_flag_name is present in feature_flag_names.
+
+        Args:
+            _: str. The id of the user.
+            target_feature_flag_name: str. The name of the target feature flag.
+
+        Returns:
+            enable_feature_flag: bool. Returns True if the target feature flag
+            name is in feature_flag_names list.
+        """
+        enable_feature_flag = False
```

Comment:
You can probably do something along the lines of `return any(...)` with a list comprehension -- I believe this shortcuts things and returns as soon as it hits something for which the condition is true.

Reply:
Done